File: socialNetwork/src/main/java/com/ftn/socialNetwork/model/entity/GroupAdmin.java
package com.ftn.socialNetwork.model.entity;

import javax.persistence.*;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;


@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "groupAdmins")
public class GroupAdmin{
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

  @JsonIgnore
    @OneToOne
    private Group group;

    @OneToOne
    private User user;
}

File: src/app/models/groupAdmin.ts
import { User } from "./user";
import { Group } from "./group";
export class GroupAdmin {
    id: number;
    group: Group;
    user: User;
  
    constructor(id: number, group: Group, user: User) {
      this.id = id;
      this.group = group;
      this.user = user;
    }
  }
  